interface.go

```diff
@@ -1,11 +1,9 @@
-package storage
+package elrond_go_storage
```

Comment:
maybe move this file as well into common?

Reply:
moved to a new package